diff --git a/contrib/loaders/flash/fpga/xilinx_bscan_spi.py b/contrib/loaders/flash/fpga/xilinx_bscan_spi.py
index 4246aa2f..f92d5d2d 100755
--- a/contrib/loaders/flash/fpga/xilinx_bscan_spi.py
+++ b/contrib/loaders/flash/fpga/xilinx_bscan_spi.py
@@ -323,7 +323,7 @@ class XilinxBscanSpi(xilinx.XilinxPlatform):
         ("fg484", 1): ["Y4", "AA20", "AB14", "AB20"],
         ("fgg484", 1): ["Y4", "AA20", "AB14", "AB20"],
         ("fgg400", 1): ["Y2", "Y19", "W12", "W18"],
-        ("ftg256", 1): ["T2", "R14", "P10", "T14"],
+        ("ftg256", 1): ["L12", None, "J13", "J14"],
         ("ft256", 1): ["T2", "R14", "P10", "T14"],
         ("fg400", 1): ["Y2", "Y19", "W12", "W18"],
         ("cs484", 1): ["U7", "V17", "V13", "W17"],
@@ -393,7 +393,7 @@ class XilinxBscanSpi(xilinx.XilinxPlatform):
         "xc7a100t": ("csg324-1", 1, "LVCMOS25", Series7),
         "xc7a15t": ("cpg236-1", 1, "LVCMOS25", Series7),
         "xc7a200t": ("fbg484-1", 1, "LVCMOS25", Series7),
-        "xc7a35t": ("cpg236-1", 1, "LVCMOS25", Series7),
+        "xc7a35t": ("ftg256", 1, "LVCMOS25", Series7),
         "xc7a50t": ("cpg236-1", 1, "LVCMOS25", Series7),
         "xc7a75t": ("csg324-1", 1, "LVCMOS25", Series7),
         "xc7k160t": ("fbg484-1", 2, "LVCMOS25", Series7),
